source/modules/assets/cartridges.rst

```diff
@@ -0,0 +1,59 @@
+Cartridges
+==========
+
+This functionality of `Asset` module allows to create cartridges models and corresponding cartridges.
+
+In a cartridge form, the following informations are available:
+
+- general characteristics of the cartridge: vendor, type, reference...
```

Comment:
For this and other lists, it may be cleaner to use nested lists.
```suggestion
- General characteristics of the cartridge:
  - Vendor
  - Type
  - Reference
```
I'm not sure the best way to indicate that these lists are not exhaustive right now.

Also, more of a general note for this PR, list items should start with a capital letter.

Reply:
Maybe just:
```suggestion
- General characteristics of the cartridge:
  - Vendor
  - Type
  - Reference
  - ...
```

Reply:
Changes done: nested lists and items starting with a capital letter